File: 1-Iniciante/1151-Fibonacci-Facil/1151.c
# include <stdio.h>

int main() {

    int atual, ant = 1, antsuc = 0, analog;
    scanf ("%d", &atual);
    analog = atual - 2;
    printf ("0 1 ");
    for (int i = 1; i <= analog; i++) {
        atual = ant + antsuc;
        antsuc = ant;
        ant = atual;
        if (i == analog) {
            printf ("%d\n", atual);
        }
        else {
            printf ("%d ", atual);
        }
    }

    return 0;
}

File: 1-Iniciante/1151-Fibonacci-Facil/1151.py
a = int(input())
num = 0
ant = 0 
antsuc = 1
print (0, "" ,end='')
for _ in range (a - 1):
    num = ant + antsuc
    if _ != a - 2:
        print (num,"",end='')
    else:
        print (num)
    antsuc = ant
    ant = num